examples/with-mongodb/lib/mongodb.ts

```diff
@@ -1,33 +1,43 @@
-import { MongoClient } from 'mongodb'
+import { MongoClient, Db } from 'mongodb';
 
-if (!process.env.MONGODB_URI) {
-  throw new Error('Invalid/Missing environment variable: "MONGODB_URI"')
+
+const MONGODB_URI = process.env.MONGODB_URI;
```

Comment:
Why do we need to have these separated?

Reply:
Did you mean `env` variables ? 
`process.env.MONGODB_DB` is optional. If not provided then it will use default one from `MONGODB_URI`.